### PR TITLE
Add padding to layout

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .header {
   padding: 8px 16px 0;
+  margin: 0 -16px;
```
Comment (iceicery) [inline src/components/Header/Header.css:3]: I am not sure why you add padding 16px and margin -16px. Why not just delete it?

**Review by iceicery — COMMENTED** [review]
Great work! I saw the layout remains the same after the change and the code is more adjustable and clean.

File: src/components/Header/Header.css (hunk 1 above)
Comment (julieg18) [reply src/components/Header/Header.css:3]: The header has a background image so it needs to stretch across the entire page, like the timeline.

**Review by iceicery — APPROVED** [review]
Approved! Great job!

**Pull request merged**